git-cheatsheet/commands.js

```diff
@@ -553,6 +561,7 @@ var translations = {
       "add -u": {"cmd": "add -u", "docs": "Adds the current content of modified (NOT NEW) files to the index.  This is similar to what 'git commit -a' does in preparation for making a commit."},
       "add -u": {"cmd": "add -u", "docs": "Añade el contenido actual de los archivos modificados (NO NUEVOS) al index. Es similar a lo que hace 'git commit -a' al prepararse para realizar un commit."},
       "rm x": {"cmd": "rm <file(s)...>", "docs": "Elimina uno o varios archivos del espacio de trabajo e index."},
+      "rm --cached x": {"cmd": "rm --cached <file(s)...>", "docs": "Elimina uno o varios archivos del espacio de trabajo e index."},
```

Comment:
This and the other translations need to be fixed to remove the workspace bit. In this case:

```suggestion
      "rm --cached x": {"cmd": "rm --cached <file(s)...>", "docs": "Elimina uno o varios archivos del index."},
```

Reply:
Add a precision: "and put it back to the workspace"?